src/elm/Page/Community/Settings/Features.elm

```diff
@@ -87,6 +91,7 @@ view loggedIn model =
                             ]
                             [ toggleView translations (translate "community.objectives.title_plural") model.hasObjectives ToggleObjectives "actions"
                             , toggleView translations (translate "menu.shop") model.hasShop ToggleShop "shop"
+                            , kycToggleView translations (translate "community.kyc.title") model.hasKyc ToggleKyc "kyc"
```

Comment:
Instead of writing a new function for toggleView that just hides the ID, the easier and more semantic way to disable this UI element would be to pass it a message that does nothing.

```suggestion
                            , toggleView translations (translate "community.kyc.title") model.hasKyc ToggleKyc "kyc"
```

See my next comment for more details on how to change ToggleKyc